gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -128,6 +129,13 @@ struct _DyldImageInfo64
   guint64 image_file_mod_date;
 };
 
+struct _gum_malloc_helper_info
```

Comment:
Oh, I missed this: structs should use CamelCase, and there should always be a typedef above it (together with the other typedefs). We typically also name these something along the lines of:
```C
struct _GumEnumerateMallocRangesContext;
```
(It might be a bit confusing because of the Info structs above, but that's because it's just mirroring some Darwin structures.)